src/libraries/Microsoft.Extensions.Logging/tests/Common/LoggerFactoryTest.cs

```diff
@@ -213,6 +267,74 @@ public void TestInvalidActivityTrackingOptions()
             );
         }
 
+        [Fact]
+        public void TestActivityTrackingOptionsBaggage()
```

Comment:
Added explicit tests for baggage and tags, to test the "empty" and null cases as well as the case when we have multiple baggages and tags (where as in the Theory we have only one tag and baggage)